pkg/operator/mon/mon.go

```diff
@@ -40,6 +40,10 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mon")
 
 const (
+	// EndpointConfigMapName is the name of the configmap with mon endpoints
+	EndpointConfigMapName = "mon-config"
```

Comment:
@jbw976 yes, the config map should be more specific since it doesn't contain any other config.
@kokhang yes, they are referenced outside the package, while none of the other are referenced externally.